core/ui/SideBar/NewTabButton.tid

```diff
@@ -0,0 +1,14 @@
+created: 20160319212903860
+modified: 20160319221316664
```

Comment:
@pmario Good catch. I've corrected the commit.

@sukima The node.js setup does have some usability issues with a git-based workflow. Another point is that paths containing `$` are a pain to work with, you have to escape them on the shell and for some reason vim's tab completion doesn't work with them. It would be much better if the filesystem backend would replace `$:/` with `.`, which avoids these issues and marks "hidden" tiddlers as such (at least on unixoid operating systems).